### DISCUSSION TITLE
v8 release estimation

### DISCUSSION THREAD

**Discussion by hi-reeve** [TanStack/table#4045] [Q&A]
glad i found this project and thankyou for saving a lot of developer time when building the most complex component.
is there any estimated time on v8 release?
because i'm using vue version and there is lack of example and the v8 api docs also not completed, so it's kinda hard, even when following the react example because of how the reactivity on vue works.
thankyou.

**Comment by antipopp** [comment]
TBF even the classic kitchen sink example would be enough to bootstrap the table in our projects, mostly to have an idea on how to combine the different features in one single table.
